src/views/Users.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button default-href="/" slot="start" />
+        <ion-title>{{ $t("Users") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button fill="clear" class="mobile-only">
+            <ion-icon slot="icon-only" :icon="filterOutline" />
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content :fullscreen="true">
+      <div class="find">
+        <section class="search">
+          <ion-searchbar :placeholder="$t('Search users')" />
+        </section>
+
+        <aside class="filters">
+          <ion-list>
+            <ion-item lines="none">
+              <ion-icon :icon="globeOutline" slot="start" />
+              <ion-label>{{ $t("Shop") }}</ion-label>
+              <ion-select value="a">
+                <ion-select-option value="a">Product store</ion-select-option>
+                <ion-select-option value="b">Product store</ion-select-option>
+                <ion-select-option value="c">Product store</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-item lines="none">
+              <ion-icon :icon="idCardOutline" slot="start" />
+              <ion-label>{{ $t("Role") }}</ion-label>
+              <ion-select value="fulfillment">
+                <ion-select-option value="fulfillment">Fulfillment manager</ion-select-option>
+                <ion-select-option value="merchendiser">Merchendiser</ion-select-option>
+                <ion-select-option value="administrator">Administrator</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-item lines="none">
+              <ion-icon :icon="toggleOutline" slot="start" />
+              <ion-label>{{ $t("Status") }}</ion-label>
+              <ion-select value="active">
+                <ion-select-option value="active">Active</ion-select-option>
+                <ion-select-option value="away">Away</ion-select-option>
+                <ion-select-option value="dnd">Do Not Disturb</ion-select-option>
+              </ion-select>
+            </ion-item>
+          </ion-list>
+        </aside>
+
+        <main>
+          <div class="list-item">
+            <ion-item lines="none" @click="() => router.push('/user')">
+              <ion-icon :icon="personCircleOutline" slot="start" />
+              <ion-label>
+                Fullname
+                <p>username</p>
+                <p>email</p>
+              </ion-label>
+            </ion-item>
+
+            <ion-label>
+              14 Jan 2021
+              <p>{{ $t("created") }}</p>
+            </ion-label>
+
+            <ion-chip outline @click="editLocation()">
+              <ion-label>4 {{ $t("locations") }}</ion-label>
+            </ion-chip>
+
+            <div class="mobile">
```

Comment:
instead of making this a mobile column I would suggest adding the mobile label for role to the last column using a div to wrap both the label and the button in display flex with a gap.

Reply:
I have removed the mobile class from this column and created a div, inside that div I have wrapped mobile-only note and button. Then I have overridden the list-item last-child display property from unset to flex because if I write the display flex property on div I created for wrapping, it will not work since the display unset property is being applied globally on the last-child and here div is the last-child. So I have overridden the last child styling on this page from unset to flex and also provided some gap.